src/Avalonia.Input/FocusManager.cs

```diff
@@ -145,7 +145,7 @@ static FocusManager()
         /// Notifies the focus manager of a change in focus scope.
         /// </summary>
         /// <param name="scope">The new focus scope.</param>
-        public void SetFocusScope(IFocusScope scope)
+        public void SetFocusScope(IFocusScope? scope)
```

Comment:
We shouldn't declare the input nullable if we just throw if it's null

Reply:
ah yes, this is a left over, I started off trying to see if api should accept null to clear focus scope and quickly realised it was a bad idea. must have left this in... good spot! That would have easily been missed.

Reply:
fixed.